lib/omni_datetime_picker.dart

```diff
@@ -16,6 +16,9 @@ import 'package:omni_datetime_picker/src/omni_datetime_range_picker.dart';
 ///
 Future<DateTime?> showOmniDateTimePicker({
   required BuildContext context,
+  String? title,
+  bool? useDivider,
+  TextStyle? titleStyle,
```

Comment:
A small suggestion, maybe switching the title to be an optional widget would allow us to both remove the `titleStyle` property and give more flexibility to the user?

Also, we can apply the same logic to the divider, and just call it a separator, maybe someone would like to use a nice image here in 80s forum style instead of a divider 😄 so passing an optional `separator` widget would give more flexibility IMHO.

```suggestion
  Widget? title,
  Widget? separator,
```